Numpy.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
```

Comment:
Code should be seperated into functions. 
Please do this so that your code can be reviewed properly.